apps/framework-cli/src/cli/routines.rs

```diff
@@ -364,7 +364,11 @@ fn start_leadership_lock_task(redis_client: Arc<Mutex<RedisClient>>, project: Ar
         loop {
             interval.tick().await;
             if let Err(e) = manage_leadership_lock(&redis_client, &project).await {
-                error!("Error managing leadership lock: {}", e);
+                if !e.to_string().contains("Failed to attempt lock") {
```

Comment:
We might want to use a const for this so whoever throws this error uses the same message. If one side changes, we'll know about the other side.

Reply:
Good call, we can update in a new PR.